pkg/provider/azure_standard.go

```diff
@@ -283,7 +283,8 @@ func (az *Cloud) getLoadBalancerRuleName(service *v1.Service, protocol v1.Protoc
 	ruleName := fmt.Sprintf("%s-%s-%d", prefix, protocol, port)
 	subnet := subnet(service)
 	if subnet == nil {
-		return getResourceByIPFamily(ruleName, isIPv6)
```

Comment:
would this block ipv6?

Reply:
This reverts the change in last dualstack PR but logic stays the same. So far, `getLoadBalancerRuleName()` returns rule name with no IP family suffix.